models/tiktok_ads__account_report.sql

```diff
@@ -0,0 +1,57 @@
+with hourly as (
+    
+    select *
+    from {{ var('ad_report_hourly') }}
+
+), advertiser as (
```

Comment:
smalllll [style](https://github.com/fivetran/dbt_style_guide/blob/main/sql_guidelines.md#ctes) change for all models -- can you put the name of the CTE on a newline as well as remove the newline before the closing parentheses?